assistant_bot/record_contact.py

```diff
@@ -14,6 +14,8 @@ def __init__(self, name):
         self.birthday: FieldBirthday = None
         self.email: FieldEmail = None
         self.address: FieldAddress = None
+        self.created = datetime.now().strftime("%d.%m.%Y %H:%M")
```

Comment:
It is worth transferring similar operations to methods record_updated() and record_crated() and calling them as necessary.